sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -310,4 +315,22 @@ export interface ApplicationData {
    * Student number.
    */
   studentNumber?: string;
+  /**
+   * Program id selected by the student.
+   */
+  selectedProgram?: number;
+  /**
+   * Program name selected by the student.
```

Comment:
Can you please adjust the comment, it does not seem accurate.
This is part of the dynamic data payload to allow the student to provide what he is looking for when the desired program is not available for selection. This will be used during the PIR process.

Reply:
extended it to a different interface